### PR TITLE
Responisive web design for desktop view

### PR DESCRIPTION
**Pull request by aeh1707**
- Breakpoints were set at 3 different width ranges to comply with most of the views, whether it's desktops or even tablets.
- Grids were used to layout cards and stack them differently, flexbox was used to arrange inner card's elements and also for contact about sections.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -330,6 +352,159 @@ textarea {
...
+    flex-direction: row-reverse;
+  }
+
+  .card-2 .card-content,
+  .card-4 .card-content {
+    margin-right: 10px;
+  }
+}
+
+@media (min-width: 900px) {
```
Comment (Ismailco) [inline styles.css:491]: - [x] The hamburger menu should be hidden in this screen size (768px or wider) and displayed when the screen size is less than 768px, kindly modify your media query width to target this sizes

File: styles.css [hunk 2]
```diff
@@ -330,6 +352,159 @@ textarea {
...
+  .card-3 {
+    grid-area: 3/2/4/4;
+  }
+
+  .card-4 {
+    grid-area: 3/4/4/6;
+  }
+}
+
+@media (min-width: 820px) {
```
Comment (Ismailco) [inline styles.css:452]: - [ ] Within each of the drop-down menus there should be up to 5 technologies per row, please implement these changes

Screenshot_20220428_154651

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @aeh1707,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (aeh1707) [reply styles.css:491]: Thanks, I just fixed it. :)

**Review by zainsadaqat — APPROVED** [review]
# Hello @aeh1707 😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**